packages/flex/src/Flex.module.css

```
@@ -0,0 +1,11 @@
+.flex {
+  display: var(--flex-display, flex);
```

```suggestion
  display: var(--flex-display);
```
컴포넌트에서 기본값 할당해주고 있으니 fallback 없어도 될 것 같아~